network-store-model/src/test/java/com/powsybl/network/store/model/ResourceTest.java

```diff
@@ -148,6 +148,7 @@ public void lineTest() throws IOException {
                         .b1(1)
                         .g2(1)
                         .b2(1)
+                        .branchStatus("IN_OPERATION")
```

Comment:
BranchStatus.Status.IN_OPERATION.name() ?

Reply:
network-store-model does not have any dependency to powsybl-single-line-diagram-iidm-extensions and it is not necessary to add one. branchStatus attribute is just a String and therefore, there is no need to ensure this value is one of the BranchStatus.Status possible values in this test.